index.js

```diff
@@ -19,12 +19,12 @@ const queue = new Map();
 bot.on("warn", console.warn);
 bot.on("error", console.error);
```

Comment:
Tell me if I also need to rename this to `shardError`

Reply:
Makasih banyak gan, niat bukan main xD